<script type="module">
  import {Assert, RR0AssertionError} from "../../dist/index.js"

  new Promise(async () => {
    const assert = await Assert.getInstance()
    assert.ok(true, "true is not true")
    console.log("ok positive")
    try {
      assert.ok(false, "false is not true")
    } catch (e) {
      if (!(e instanceof RR0AssertionError)) {
        throw e
      }
      console.log("ok negative")
    }
  })
</script>
